modules/ROOT/pages/sending-data-from-arm-to-external-analytics-software.adoc

```diff
@@ -158,321 +102,259 @@ INDEXED_EXTRACTIONS = JSON
+. Select the type of information to send from the *Level* menu.
 +
-image::sending-data-from-arm-to-external-monitoring-software-level.png[]
-
-. On the *Event Tracking* region, activate the *Splunk* switch. A pop-up menu opens where you can provide your Splunk user and password data, as well as the host and port for the connection.
+See <<Logging Level Options>>.
+. Under *Event Tracking*, enable *ELK*.
+. Click the gear icon to configure the log file location for ELK:
 +
-image::sending-data-from-arm-to-external-monitoring-software-splunk.png[]
+.The screenshot shows (*1*) the log file location and (*2*) the *Show more options* switch in the ELK configuration window.
+image::hybrid-export-elk-configure.png[Log file location and Show more options switch in the ELK configuration window]
 
-. Activate the *Splunk* switch to open a pop-up menu. In the Dropdown pick *TCP*, then provide the host and port for the connection.
+. If you want, click *Show more options* to specify additional properties for the log file:
 +
-image::agent-to-splunk-tcp.png[splunk]
+.The arrow shows the expanded options in the ELK configuration window.
+image::hybrid-export-elk-configure-advanced.png[Expanded options in the ELK configuration window]
 
-==== Integrating with an ELK Stack
 
-ELK combines three open source tools (Elasticsearch, Logstash, Kibana) that work together to help you store, search and analyze log data. You can output the Mule event notifications as generic system logs, which can be handled by your ELK stack. Logstash captures and indexes the data into the log, from which you can then use Elasticsearch and Kibana to generate graphs, reports, alerts, dashboards and visualizations.
-The Runtime Manager agent helps you store all of the Event Notifications produced from the Mule runtime flows into a configurable log file with a rolling file policy.
+. Click *Apply* to save your changes.
 
-image::amc-onprem-diagram-detail-elk.jpg[]
 
-To direct information to the folder where your ELK stack reads from, you must do the following:
+== Integrating API Analytics with Splunk or ELK
 
-. Select the server whose information you want to send out.
-. In the menu on the right, click *Manage Server* to access the server settings.
-+
-image::sending-data-from-arm-to-external-monitoring-software-manage-server.png[]
+To integrate API Analytics with Splunk or ELK:
 
-. Select the *Plugins* tab:
-+
-image::sending-data-from-arm-to-external-monitoring-software-plugins.png[]
+. <<Configure the API Gateway>>.
+. <<Configure API Analytics to Connect with Splunk or ELK>>.
 
-. Select the kind of information that you want to send out in the *Level* dropdown menu
-+
-image::sending-data-from-arm-to-external-monitoring-software-level.png[]
+[WARNING]
+If you configure the API gateway `wrapper.conf` (step 1) but don't assign an external destination for your data (step 2), the analytics data accumulates in a queue in the API gateway server, which could cause the system to crash.
```

Comment:
```suggestion
If you configure the API gateway `wrapper.conf` (Step 1) but don't assign an external destination for your data (Step 2), the analytics data accumulates in a queue in the API gateway server, which could cause the system to crash.
```

Reply:
Should this warning appear with the previous use of these same steps?

Reply:
No - the steps are different. This applies to the API Gateway bit. For the other, the change is on the Splunk side.